ruby/example_code/dynamodb/GettingStarted/MoviesCreateTable.rb

```diff
@@ -0,0 +1,72 @@
+# snippet-sourcedescription:[MoviesCreateTable.rb demonstrates how to ]
+# snippet-service:[dynamodb]
+# snippet-keyword:[Ruby]
+# snippet-keyword:[Amazon DynamoDB]
+# snippet-keyword:[Code Sample]
+# snippet-keyword:[ ]
+# snippet-sourcetype:[full-example]
+# snippet-sourcedate:[ ]
+# snippet-sourceauthor:[AWS]
+# snippet-start:[dynamodb.Ruby.CodeExample.MoviesCreateTable] 
```

Comment:
Move snippet-start to just before code. Otherwise the example will show the copyright in the docs, which is not necessary and will clutter up the docs.